docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -1,91 +1,83 @@
 [id='{context}-pro-running-the-demo-repositories']
 = Running the {PROJECT_NAME} Applications on a Local Machine
 
-The {PROJECT_NAME} {PROJECT_NAME} applications in this section are dependent on the _public NPM Registry_.
-
-This allows for a simpler setup of the {PROJECT_NAME} applications for users that only require the application to run, and will not engage in development.
+Get started by running {PROJECT_NAME} applications on your local machine. Follow
+these steps and use the _public NPM Registry_ to set up the {PROJECT_NAME}
+applications that contain sample Steps, Workflows, and Workorders.
 
 [discrete]
-== Prerequisites
-
-Ensure you have:
+== Requirements
 
-* Node and npm installed (tested on Node {WFM-RC-NodeVersion})
+The following must be installed on your system:
+* Node and npm (tested version: {WFM-RC-NodeVersion})
 * git
-* access to `git@github.com:feedhenry-raincatcher/raincatcher-server.git`
-* access to `git@github.com:feedhenry-raincatcher/raincatcher-mobile.git`
-* access to `git@github.com:feedhenry-raincatcher/raincatcher-portal.git`
-* MongoDB (tested with version: {WFM-RC-MongoDBVersion})
-* Redis (tested with version: {WFM-RC-RedisVersion})
+* MongoDB
+* Redis
 
-[id='{context}-published-repositories-cloning-the-git-repositories']
-[discrete]
-== Cloning the Git Repositories
+You must have access to GitHub.
 
-The following commands will clone the `master` branch of each repository and they collectively contain the latest tagged release of the {PROJECT_NAME} project.
+[discrete]
+== Installing and Running the {PROJECT_NAME} Applications
 
+1&#46; Clone the Git repositories.
 [source,bash]
 ----
 git clone git@github.com:feedhenry-raincatcher/raincatcher-server.git
 git clone git@github.com:feedhenry-raincatcher/raincatcher-mobile.git
 git clone git@github.com:feedhenry-raincatcher/raincatcher-portal.git
 ----
 
-Previous versions are available through branches named with the `release-\{semver\}` convention.
-For example, in order to checkout version `0.0.1` of `raincatcher-server`, run the following commands after the cloning:
+The master branch of each Git repository provides the latest version. Previous
+versions have the `release-*` tag. For example, run the following command to
+checkout version `1.0.0`:
 
 [source,bash]
 ----
-cd raincatcher-server/
-git checkout release-0.0.1
+cd /raincatcher-server
+git checkout release-1.0.0
 ----
 
-[id='{context}-published-repositories-installing-repository-dependencies']
-[discrete]
-== Installing Repository Dependencies
-
-For each repository previously cloned, run `npm install` so regular NPM dependencies are downloaded.
-
-[id='{context}-published-repositories-starting-mongodb-and-redis']
-[discrete]
-== Starting MongoDB and Redis
+2&#46; Download the NPM dependencies for each Git repostory.
```

Comment:
Why we have this? Can we revert to original `==` context here is important as it's referenced in the other file.

Reply:
hey Wojciech, sorry for not explaining why I did this. I actually meant to post a comment to discuss the contexts.

honestly I missed the step to do 'npm install' when I went through the procedure. I think having separate headings for each step adds clutter and makes it more complex. one thing I've noticed in the docs is that it's hard to figure out what's going on at a high-level. I really find some of the headings are confusing.

I've been doing some more work on other parts of the doc. I'd actually like to spend a bit more time on all the content. as part of that, I'd also like to reduce references between files. that can add complexity and maintenance overhead, which is better to avoid imo.

I don't mean to be disruptive to the project or annoy you with lots of unnecessary edits. documentation PRs can often be harder to manage and review than code changes, which can be smaller and more discrete. what would you think if I spent more time and committed more changes with the goal of simplifying the docs and making them more concise?

or would it be helpful if I just looked more for minor edits and issues, rather than making structural changes?

please just let me know what I can do to help. I'm happy to contribute much more time to the docs.

btw, this is cool tech. it's been fun learning so far.

cheers!

Reply:
@oraNod Absolutely no problem. Your changes are brilliant. 
In this particular case we need to be really cautious as we using headings for all procedures. If we change this we need to make sure that others will change and future content will also follow that pattern. iIf we drop email to email list with some ideas.

> I don't mean to be disruptive to the project or annoy you with lots of unnecessary edits. 

Not at all. Your changes are solid and improving things. Try to avoid changes that will have more global scope - like overall style changes etc. 

PR's should always have narrow scope and resolve specific problem. It's also good to have issue logged in but that's not required. If you have any suggestions drop us an email. 

> or would it be helpful if I just looked more for minor edits and issues, rather than making structural changes?

Project is open for any type of input, but what you looking overall is to have your PR merged. If you will clearly separate your efforts into small PR's and do conceptual/global changes in other PR's this will help a lots with reviews. I see a lots of good content in current PR but we cannot merge it as this new format is breaking webpage build (we using asciidoctor for builds).

Suggestion is to revert this changes and open conversation on the email group.

Reply:
@wtrocki sounds good. yeah, this PR has started to expand out. I'll revert the header changes and keep the original structure. I'll get in touch  on the email group and we can take it from there.

cheers.